### PR TITLE
Add notes for 2022-07

### PR DESCRIPTION
**Pull request by robpalme**
ref: https://github.com/tc39/Reflector/issues/437
ref: https://github.com/tc39/Reflector/issues/442

### CODE REVIEW

File: meetings/2022-07/jul-21.md [hunk 1]
```diff
@@ -0,0 +1,795 @@
...
+### Conclusion/Resolution
+
+- Stage 1
+
+## Ergonomic Dynamic Object Restructuring
+
+Presenter: Hemanth HM (HHM)
+
+- [proposal](https://github.com/tc39-transfer/proposal-object-pick-or-omit)
+- [slides]()
```
Comment (dbartholomae) [inline meetings/2022-07/jul-21.md:566]: It seems the link to the slides is missing

**Comment by michaelficarra** [comment]
Shouldn't the notes from the transcriptionist be converted to markdown?

**Comment by jmdyck** [comment]
I've submitted a PR against this PR's branch.

File: meetings/2022-07/jul-21.md (hunk 1 above)
Comment (robpalme) [reply meetings/2022-07/jul-21.md:566]: There was no slide link on the agenda. This is for @hemanth to fix up later.

**Review by ljharb — APPROVED** [review]

**Pull request merged**